modules/application-health-configuring.adoc

```diff
@@ -5,7 +5,7 @@
 [id="application-health-configuring_{context}"]
 = Configuring health checks
 
-To configure readiness, liveness, and startup probes, add one or more probes to the specification for the pod that contains the container which you want to perform the health checks
+To configure readiness and liveness probes, add one or more probes to the specification for the pod that contains the container which you want to perform the health checks
```

Comment:
s/which/that
also missing period

Also, this sentence is a bit unwieldy. I don't really understand "the pod that contains the container that you want to perform the health checks". Suggested rewrite to something like:

"To configure readiness and liveness probes, add one or more probes to the pod specification for the container that you want to perform the health checks on." (Not sure if that retains the meaning.)